test_ak_survey_results.py

```diff
@@ -1,20 +1,45 @@
 import pytest
-import test_settings
-from ak_survey_results import AKSurveyResults
+import boto3
+import json
+from moto import mock_secretsmanager
+from pywell.secrets_manager import get_secret
 from ak_survey_results import PageNotFoundException
 from ak_survey_results import PageNotSurveyException
 from ak_survey_results import PageNotLoadedException
 from ak_survey_results import Struct
 
 
+mock_secret = {'DB_SCHEMA_AK': 'ak', 'DB_SCHEMA_SURVEY': 'survey_results', 'DB_TYPE': 'PostgreSQL', 'COLUMN_EXCLUDES': ''}
+redshift_secret={'username':'postgres','password':'','host':'localhost','port':'5432', 'dbName': 'postgres'}
+
+class ArgsObject:
+    def __init__(self, dict_args):
+        for arg, val in dict_args.items():
+            setattr(self, arg, val)
+
+@mock_secretsmanager
+def init_secrets():
+    client = boto3.client('secretsmanager')
+    client.create_secret(Name='ak-survey-results', SecretString=json.dumps(mock_secret))
+    client.create_secret(Name='redshift-admin', SecretString=json.dumps(redshift_secret))
+
+
+@mock_secretsmanager
 class Test:
 
-    def setup(self):
-        self.survey_results = AKSurveyResults(test_settings)
+    def setup_method(self, method):
+        init_secrets()
+        self.args = get_secret('ak-survey-results')
```

Comment:
I tested the mock by renaming the secrets `ak-survey-results-alex-fake` and `redshift-admin-alex-fake` both in the tests and in the real code, and I got an error, `botocore.errorfactory.ResourceNotFoundException: An error occurred (ResourceNotFoundException) when calling the GetSecretValue operation: Secrets Manager can't find the specified secret.`. I think the problem is with the first secret, `ak-survey-results-alex-fake`. At first, I thought it was because the call to `get_settings` is made at the global scope in ak_survey_results.py, but the same error occurred even when I moved the call into the `AKSurveyResults` `__init__` class, so there might be another detail I'm missing. Do you run into the same issue?

Reply:
It shouldn't be the global scope, fundraising-parsons-scripts has global scope too.  I'll take a look.